extensions/amp-subscriptions/0.1/score-factors.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * Copyright 2018 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Possible score factors.
+ * @const @enum {string}
+ */
+export const ampSubscriptionsScoreFactor = {
```

Comment:
Nit: canonically enums should be capitalized, e.g. `AmpSubscriptionsScoreFactor`. But overall, I'm not a big fun of `Amp` prefix in var/class/enum names.